src/coreclr/jit/emitarm64sve.cpp

```diff
@@ -9903,17 +9897,17 @@ BYTE* emitter::emitOutput_InstrSve(BYTE* dst, instrDesc* id)
     switch (fmt)
     {
         // Scalable.
-        case IF_SVE_AA_3A: // ........xx...... ...gggmmmmmddddd -- SVE bitwise logical operations + integer add/subtract
-                           // vectors + SVE integer min/max/difference (predicated)
+        case IF_SVE_AA_3A: // ........xx...... ...gggmmmmmddddd -- SVE bitwise logical operations + SVE integer
```

Comment:
i would do this change at the definition and just remove the description from here (for the ones you have already merged).

Reply:
Ok, so just remove these descriptions for the ones we merged, got it.